cmd/collector/app/span_processor_test.go

```diff
@@ -316,49 +316,52 @@ func TestSpanProcessorBusy(t *testing.T) {
 	assert.Nil(t, res)
 }
 
-func TestSpanProcessorWithNilProcess(t *testing.T) {
```

Comment:
Why was this test removed? Is the previous condition no longer true?

Reply:
Sorry. I removed it unintentionally  when copying code. It's fixed now.